src/01-general/users.apib

```diff
@@ -40,6 +40,8 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
+                + time_tracking_freezing: true (boolean)
```

Comment:
Q: can we make this one (object) property instead of two related ones?

Reply:
Yup, I would go for time_tracking_freezing -> enabled, period (optional). And period would be a value/unit object.

Reply:
+1 ☝️